notebooks/DataFrame_Widget_Demo.ipynb

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
<p>Can you add outputs to this notebook so we can compare them during prerelease testing?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Added outputs and addressed formatting